man/fi_poll.3.md

```diff
@@ -159,9 +170,10 @@ struct fi_wait_attr {
 
 - *FI_WAIT_FD*
 : Indicates that the wait set should use a file descriptor as its wait
-  mechanism.  A file descriptor wait object must be usable in select,
-  poll, and epoll routines.  However, a provider may signal an FD wait
-  object by marking it as readable, writable, or with an error.
+  mechanism. Since it may not always be possible to use a single underlying
+  wait object, the file descriptor is not guaranteed to be usable in select,
+  poll, and epoll routines. However, when possible, a provider may signal an FD
+  wait object by marking it as readable, writable, or with an error.
```

Comment:
What about this wording instead?

Indicates that the wait set should use file descriptor(s) as its wait mechanism.  It may not always be possible for a wait set to be implemented to use a single underlying file descriptor, but all wait objects will be file descriptors.  File descriptor wait objects must be usable in select, poll, and epoll routines.  However, a provider may signal an FD wait object by marking it as readable, writable, or with an error.